config/user.m4

```diff
@@ -17,15 +17,18 @@ AC_DEFUN([ZFS_AC_CONFIG_USER], [
 		ZFS_AC_CONFIG_USER_LIBUDEV
 		ZFS_AC_CONFIG_USER_LIBUUID
 		ZFS_AC_CONFIG_USER_LIBBLKID
+		ZFS_AC_CONFIG_USER_PAM
 	])
 	ZFS_AC_CONFIG_USER_LIBTIRPC
 	ZFS_AC_CONFIG_USER_LIBCRYPTO
 	ZFS_AC_CONFIG_USER_LIBAIO
 	ZFS_AC_CONFIG_USER_LIBATOMIC
 	ZFS_AC_CONFIG_USER_LIBFETCH
 	ZFS_AC_CONFIG_USER_AIO_H
-	ZFS_AC_CONFIG_USER_CLOCK_GETTIME
-	ZFS_AC_CONFIG_USER_PAM
+	AM_COND_IF([BUILD_MACOS], [
```

Comment:
shouldn't ZFS_AC_CONFIG_USER_CLOCK_GETTIME and ZFS_AC_CONFIG_USER_PAM be outside this conditional?

Reply:
If macos, run `ZFS_AC_MACOS_IMPURE_ENABLE`
else if not macos, run `ZFS_AC_CONFIG_USER_CLOCK_GETTIME`

Reply:
You've moved ZFS_AC_CONFIG_USER_PAM into the linux conditional only. what about freebsd?